lnwallet/interface_test.go

```diff
@@ -1233,6 +1238,141 @@ func testSignOutputPrivateTweak(r *rpctest.Harness, w *lnwallet.LightningWallet,
 	}
 }
 
+func testReorgWalletBalance(r *rpctest.Harness, w *lnwallet.LightningWallet, t *testing.T) {
+	t.Logf("Running reorg wallet balance test")
```

Comment:
With the latest change to the way the these tests are run, this logging statement is no longer needed. When you rebase to master, you'll instead tag a name onto this test case at the bottom of this file.